flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/utils/UserDefinedFunctionUtils.scala

```diff
@@ -324,4 +325,19 @@ object UserDefinedFunctionUtils {
     candidate == classOf[Time] && (expected == classOf[Int] || expected == classOf[JInt]) ||
     candidate == classOf[Timestamp] && (expected == classOf[Long] || expected == classOf[JLong])
 
+  @throws[Exception]
+  def serialize(function: UserDefinedFunction): String = {
+    val byteArrayOutPut = new ByteArrayOutputStream
+    val objectOutPut = new ObjectOutputStream(byteArrayOutPut)
+    objectOutPut.writeObject(function)
+    objectOutPut.close()
+    Base64.encodeBase64URLSafeString(byteArrayOutPut.toByteArray)
```

Comment:
No. we can not use that method. because:
encodeBase64String  will `Encodes binary data using the base64 algorithm into 76 character blocks separated by CRLF.` So，if the binary data more than 76 character, we'll got `CRLF` which we do not want.